tests/python/unittest/test_sparse_operator.py

```diff
@@ -1853,152 +1853,6 @@ def check_broadcast_div(mx_lhs, mx_rhs, np_lhs, np_rhs, dtype):
             check_broadcast_mul(mx_lhs, mx_rhs, np_lhs, np_rhs, np.float32)
             check_broadcast_div(mx_lhs, mx_rhs, np_lhs, np_rhs, np.float32)
 
-@with_seed()
-def test_scatter_ops():
```

Comment:
cc @eric-haibin-lin to confirm

Reply:
Those scatter ops were used long ago - introduced in #7903 to support sparse adagrad I think and then replaced in different implementation of adagrad in #9651. Still, the ops stayed, and since they were always in `_internal` space, I removed them instead of reworking them to RTC.